src/main/webapp/app/iris/course-chatbot/course-chatbot.component.ts

```diff
@@ -1,19 +1,21 @@
-import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { Component, effect, input } from '@angular/core';
 import { ChatServiceMode, IrisChatService } from 'app/iris/iris-chat.service';
+import { IrisModule } from 'app/iris/iris.module';
+import { IrisBaseChatbotComponent } from 'app/iris/base-chatbot/iris-base-chatbot.component';
 
 @Component({
     selector: 'jhi-course-chatbot',
     templateUrl: './course-chatbot.component.html',
     styleUrl: './course-chatbot.component.scss',
+    standalone: true,
```

Comment:
```suggestion
    changeDetection: ChangeDetectionStrategy.OnPush,
    standalone: true,
```
Here as well, is there a particular reason that we avoid OnPush change detection strategy here different from other components?